plugins/inputs/win_perf_counters/pdh.go

```diff
@@ -544,20 +545,20 @@ func PdhExpandWildCardPath(szWildCardPath string, mszExpandedPathList *uint16, p
 	ptxt, _ := syscall.UTF16PtrFromString(szWildCardPath)
 	flags := uint32(0) // expand instances and counters
 	ret, _, _ := pdh_ExpandWildCardPathW.Call(
-		uintptr(unsafe.Pointer(nil)), // search counters on local computer
-		uintptr(unsafe.Pointer(ptxt)),
-		uintptr(unsafe.Pointer(mszExpandedPathList)),
-		uintptr(unsafe.Pointer(pcchPathListLength)),
-		uintptr(unsafe.Pointer(&flags)))
+		// search counters on local computer
+		uintptr(unsafe.Pointer(nil)),                 //nolint:gosec // G103: Valid use of unsafe call to pass nil
```

Comment:
Is this actually needed?

Reply:
@Hipska without this conversion we have:
```
plugins/inputs/win_perf_counters/pdh.go:549:11: cannot convert nil to type uintptr
```

![image](https://user-images.githubusercontent.com/17336997/231120882-0d886ee6-93c9-4a48-a10c-107455d1cc00.png)

Reply:
As far as I can see `uintptr(unsafe.Pointer(nil))` is just `0x0`? So, might just be this?
```suggestion
		0,
```

Reply:
I'm ready to hit merge, but do we want to keep this as-is or change it?

Reply:
@Hipska @powersj Changed.